src/modules/pos/reward/hooks/queries/useValidatorRewardsWithToken.test.js

```diff
@@ -0,0 +1,28 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { queryWrapper as wrapper } from 'src/utils/test/queryWrapper';
+import { useCurrentAccount } from '@account/hooks';
+import mockSavedAccounts from '@tests/fixtures/accounts';
+import { useCommandSchema } from '@network/hooks';
+import { mockCommandParametersSchemas } from '@common/__fixtures__';
+import {useValidatorRewardsWithToken} from "@pos/reward/hooks/queries/useValidatorRewardsWithToken";
+
+jest.useRealTimers();
+
+jest.mock('@account/hooks/useCurrentAccount');
+jest.mock('@network/hooks/useCommandsSchema');
+
+useCurrentAccount.mockReturnValue([mockSavedAccounts[0]]);
+useCommandSchema.mockReturnValue({
+  moduleCommandSchemas: mockCommandParametersSchemas.data.commands.reduce(
+    (result, { moduleCommand, schema }) => ({ ...result, [moduleCommand]: schema }),
+    {}
+  ),
+});
+
+describe('useValidatorRewardsWithToken', () => {
+  it('Should toggle load when finished loading validators', async () => {
+    const { result, waitFor } = renderHook(() => useValidatorRewardsWithToken(), { wrapper });
+    await waitFor(() => result.current.isLoading === false);
```

Comment:
`await waitFor(() => !result.current.isLoading)` is another way this could have been implemented but this is fine

Reply:
Since we initialise isLoading to null `const [isLoading, setIsLoading] = useState(null);` And when we start to fetch we set value to True and then when fetched is complete we set it to False. We need to explicitly check for false since !null is true so it would never wait for the complete fetch.

Sure it could probably be done if rewriting the hook a bit, but for now isLoading is
null: when hook is first initialized
true: when hook is loading
false: when hook has finished loading (not `null`)